### ISSUE TITLE
Vox-Pidgin Exclusivity

### ISSUE THREAD

**Issue by Davnik** [HeavensGate/Eternal#197]
It is always an immersion-breaker to have a player who speaks Vox-Pidgin. While it is supposed to be allowed for the rare organism who chooses to restructure his entire mouth-to-chest region into a freaking bird, it's fair to say nobody is going to bring in that harrowed snowflake, so we're better off preventing non-vox from speaking Vox-Pidgin. 

Prevent non-vox from speaking Vox-Pidgin. Allow non-vox to specialize in Vox-Pidgin, but only allow them to understand it. Despite the unintelligibility of it all, it is just a coherent pattern of guttural birdsong. Make sure each of these characters knows the backstory implications of understanding Vox-Pidgin; it requires extensive time spent with a rare and obscure race, learning from their paranoid and xenophobic scholars, who would not want to share their difficult language with an outsider. This is the basic description of a whitelist, but we'll leave it on a blacklist the admins are expected to manage regularly.

**Comment by Jacquar** [comment]
Agreed. Alternative options - might we create a non antag mask slot item held by the hop and cmo that translates into a mechanically distorted version of the language? A rough mechanical one way babel fish?

**Comment by Davnik** [comment]
Or, babble fish, considering we're discussing vox.

**Comment by Davnik** [comment]
Meanwhile, this conversation has stirred. Apparently not even cunning linguists can make sense of Vox-Pidgin, so perhaps only Vox should be able to even understand it, much less speak it.

**Comment by Mewchild** [comment]
Direct wiki copy/paste: 
Vox-pidgin itself is bewilderingly complex and defies any concerted study by linguists.
It would be impossible for a human (or humanoid) to speak Vox-pidgin without drastic and disfiguring throat and facial surgery.

**Comment by staticcharged** [comment]
Could we do it through the pAI? They can already do translate features, what about talking through them to speak Vox?

**Comment by Davnik** [comment]
100%, beyond any debate, vox-pidgin needs to be removed from the player race language list, and forced to only apply to Vox, like Rootspeak or Changeling Hivemind. I believe the tag is... "Restricted" rather than "Whitelisted", if that helps.

**Comment by killer99603** [comment]
I'll take it out. I'm leaving this issue open though because it has ideas that are not just removing the language entirely.

**Comment by killer99603** [comment]
Just as a note, thus far, there is nothing in the Vox-pidgin code that should allow it to be chosen. It's got the same flags as binary and rootspeak. I'll keep digging into other files to see where the problem is.
